public/components/health-check/components/check-result.tsx

```diff
@@ -112,21 +150,73 @@ export function CheckResult(props) {
                 aria-label="Next"
               />
             </EuiToolTip>
-          </span>
+          </>
         );
       case 'waiting':
         return (
-          <span>
-            Waiting...
-          </span>
+          <EuiToolTip
+          position='top'
+          content='Waiting&hellip;'>
+            <EuiIcon aria-label={result} type="clock" color="#999999"></EuiIcon> 
+          </EuiToolTip>
         );
     }
   };
 
+  const tooltipVerboseButton = `${verboseIsOpen ? 'Close' : 'Open'} details`;
+  const checkDidSomeAction = useMemo(() => verboseInfo.some(log => log.type === 'action'), [verboseInfo]);
+  const shouldShowNotification = checkDidSomeAction && !verboseInfoWithNotificationWasOpened.current;
+  const haveLogs = verboseInfo.length > 0;
+  
+  const switchVerboseDetails = useCallback(() => {
+    if(checkDidSomeAction && !verboseInfoWithNotificationWasOpened.current){
+      verboseInfoWithNotificationWasOpened.current = true;
+    };
+    setVerboseIsOpen(state => !state);
+  },[checkDidSomeAction]);
+
+  const inspectLogsButton = isCheckStarted && haveLogs ? 
```

Comment:
pls move the button and logs component to separate child components in order to keep a short class

Reply:
Great idea, already split the components. Please review them!